### PR TITLE
Sample: Incoming activity to JavaScript event

### LINKED ISSUE #1516: Sample: Incoming activity to emit using JavaScript event listener
> Similar to https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/14.customization-piping-to-redux.

## Goal

Incoming activity from Web Chat will be propagated to `window.addEventListener('webchatactivity')` so devs without React/Redux knowledge can copy this short middleware and they can start playing with it.

> This could be a trimmed down version of [sample 11](https://github.com/Microsoft/BotFramework-WebChat/blob/master/samples/11.customization-redux-actions/index.html) and add a JavaScript event.

### PR DESCRIPTION
**Pull request by compulim**
> Fix #1516 

In console log, this sample will dump the activity.

image

## Changelog

- UI: [Incoming activity to JavaScript event](https://microsoft.github.io/BotFramework-WebChat/15.b.incoming-activity-event/), in [#1567](https://github.com/Microsoft/BotFramework-WebChat/pull/1567)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,7 +233,8 @@ npm run prepublishOnly
...
+| [`15.a.backchannel-piggyback-on-outgoing-activities`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/15.a.backchannel-piggyback-on-outgoing-activities)             | Advanced tutorial: Demonstrates how to add custom data to every outgoing activities.                                                                                                                                                | [Demo](https://microsoft.github.io/BotFramework-WebChat/15.a.backchannel-piggyback-on-outgoing-activities)           |
+| [`15.b.incoming-activity-event`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/15.a.incoming-activity-event)                                                       | Advanced tutorial: Demonstrates how to forward all incoming activities to JavaScript event for further processing.                                                                                                                  | [Demo](https://microsoft.github.io/BotFramework-WebChat/15.a.incoming-activity-event)                                |
```
Comment (corinagum) [inline README.md]: *a JavaScript event

File: samples/15.b.incoming-activity-event/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Sample - Forward all incoming activity to JavaScript event
+
+Web Chat client that will forward all incoming activity to JavaScript event named `webchatincomingactivity`.
```
Comment (corinagum) [inline samples/15.b.incoming-activity-event/README.md]: incoming *activities* to *a* JavaScript event

File: samples/15.b.incoming-activity-event/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# Test out the hosted sample
+
+- [Try out MockBot](https://microsoft.github.io/BotFramework-WebChat/15.b.incoming-activity-event)
+
+# Things to try out
+
+- Open development console in your browser
```
Comment (corinagum) [inline samples/15.b.incoming-activity-event/README.md]: Should we stay consistent and use the same text for every sample?

File: samples/15.b.incoming-activity-event/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- Type "help" in the send box
+- In the console log, you will see bunches of incoming activities sent from the bot
+
+# Further reading
+
+## Related samples
+
+[`15.a.backchannel-piggyback-on-outgoing-activities`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/15.a.backchannel-piggyback-on-outgoing-activities) is a sample which will add custom data to every outgoing activities.
```
Comment (corinagum) [inline samples/15.b.incoming-activity-event/README.md]: sample *that* will add custom data to every outgoing *activity*

File: samples/15.b.incoming-activity-event/index.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+              window.dispatchEvent(event);
+            }
+
+            return next(action);
+          }
+        );
+
+        window.WebChat.renderWebChat({
+          directLine: window.WebChat.createDirectLine({ token }),
+          // We will use a custom version of Redux store, which we added middleware to handle all incoming activities.
```
Comment (corinagum) [inline samples/15.b.incoming-activity-event/index.html]: store, *in* which

File: samples/15.b.incoming-activity-event/README.md (hunk 3 above)
Comment (compulim) [reply samples/15.b.incoming-activity-event/README.md]: Yes, let's make a PR to scrub all the text in README.

**Pull request merged**